awscli/customizations/s3/subcommands.py

```diff
@@ -369,6 +369,11 @@
                         'output is suppressed.')}
 
 
+NO_PROGRESS = {'name': 'no-progress', 'action': 'store_true',
+                    'help_text': (
+                        'File transfer progress is not displayed.')}
```

Comment:
It's probably worth documenting that this arg only matter is the other two options aren't provided (quiet/only_show_errors)